Source/WebKit/UIProcess/WebAuthentication/fido/CtapAuthenticator.cpp

```diff
@@ -179,6 +180,7 @@ void CtapAuthenticator::continueMakeCredentialAfterResponseReceived(Vector<uint8
 
 void CtapAuthenticator::getAssertion()
 {
+    driver().setMaxMsgSize(m_info.maxMsgSize());
```

Comment:
Let's set this in `FidoService::continueAfterGetInfo(` right before we call `CtapAuthenticator::create.` You will be able to call `driver->setMaxMsgSize(info->maxMsgSize())` there. 

You don't need to worry about the U2F case because U2F messages have a strict upper bound and also won't have a `AuthenticatorGetInfoResponse`.